.drone.yml

```diff
@@ -246,34 +246,34 @@ pipeline:
       event: tag
       status: success
 
-  publish-vic-machine-server-dev:
-    image: plugins/gcr
+  vic-machine-server-tags:
+    image: '${TEST_BUILD_IMAGE=gcr.io/eminent-nation-87317/vic-integration-test:1.48}'
     pull: true
-    repo: eminent-nation-87317/vic-machine-server
-    dockerfile: cmd/vic-machine-server/Dockerfile
-    custom_dns: 10.142.7.21, 10.142.7.22, 10.166.17.90
-    secrets: [ token ]
-    tags:
-      - dev
+    environment:
+      BIN: bin
+      SHELL: /bin/bash
+    commands:
+      - 'echo -n "$(git describe --tags --abbrev=0)-${DRONE_BUILD_NUMBER}-$(git rev-parse --short=8 HEAD)" > .tags;'
+      - 'if [ "${DRONE_BUILD_EVENT}" = "push" -a "${DRONE_BRANCH}" = "master" ];     then echo ",dev" >> .tags; fi'
```

Comment:
Why four spaces in `; ﻿ ﻿ ﻿ then`? (It seems like you'd either want 1 space or enough to vertically align with the line below.)

Reply:
I will reformat this. It was meant to be formatted with a line that no longer exists.